salt/states/file.py

```diff
@@ -3643,9 +3643,11 @@ def recurse(name,
           - include_pat: E@hello      :: regexp matches 'otherhello',
                                          'hello01' ...
 
+        .. versionchanged:: Neon
```

Comment:
Please add on a separate line, indented 4 spaces from the `.. versionchanged::`, an explanation of _what_ changed (e.g. "A list of patterns is now supported.")

It would also be helpful to include some SLS examples here in the docstring.

Reply:
Done!